testing/amp-performance-extension/measure.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function renderMeasurement(container, label, count) {
+  const line = document.createElement('div');
+  line.classList.add('i-amphtml-performance-line');
+  const labelSpan = document.createElement('div');
+  labelSpan.classList.add('i-amphtml-performance-label');
+  labelSpan./* OK */ innerText = label;
+  const countSpan = document.createElement('div');
+  countSpan.classList.add('i-amphtml-performance-count');
+  countSpan./* OK */ innerText = count;
+  line.appendChild(labelSpan);
+  line.appendChild(countSpan);
+  container.appendChild(line);
+  return countSpan;
+}
+
+function addStyleString(root, str) {
+  const node = document.createElement('style');
+  node./*OK*/ innerHTML = str;
+  root.appendChild(node);
+}
+
+function round(num, dec = 4) {
+  return Math.round(num * Math.pow(10, dec)) / Math.pow(10, dec);
```

Comment:
Is rounding necessary? Due to floating point division, you're probably not going to get a rounded number anyways. We can truncate with `toFixed(decimals)`